plugins/react.js

```diff
@@ -19,6 +19,13 @@ module.exports = {
     }
   },
   rules: {
+    // enforces using semantic DOM elements over the ARIA role property.
+    'jsx-a11y/prefer-tag-over-role': 0,
```

Comment:
This rule is important to Garden and should be set to `2` error due to the [first rule of ARIA](https://www.w3.org/TR/using-aria/#rule1)